ebpfcore/ebpf_drv.c

```diff
@@ -303,6 +303,8 @@ _ebpf_driver_io_device_control(
                         goto Done;
                     }
                     user_reply = output_buffer;
+                    user_reply->length = (uint16_t)actual_output_length;
```

Comment:
Fairly sure this won't work as writing to user_reply->length will overwrite user_request->length because output_buffer == input_buffer.

Reply:
Rewrote logic